src/routes/user.ts

```diff
@@ -0,0 +1,28 @@
+import express from 'express';
+import { findLatestMetadataByProperty, findPagesByUser } from '../services/database';
+import { streamData } from '../services/dataStreamer';
+import { getRequestId } from '../utils';
+
+const router = express.Router();
+
+// Returns all build data needed for all branches of all projects for a single github user
+router.get('/:githubUser', async (req, res, next) => {
```

Comment:
Should we include `/documents` at the end of the endpoint to denote this endpoint is specifically for build documents/data?

Reply:
We could! I didn't because it already mentions a username, so it's not attempting to get the user. But maybe there's a future case I'm not thinking of. Idk if it's necessary, but it could provide clarity. This one's up to you! Easy to do

Reply:
I was gonna say we could add it just for consistency, and if we ever need to refactor the `/documents` endpoints, it'll be easy to identify all of them. Future use cases are also definitely possible considering the nature of how much this API is changing!